.circleci/config.yml

```diff
@@ -27,7 +27,7 @@ parameters:
 
   node-version:
     type: string
-    default: 20.9-browsers
+    default: 20.11-browsers
```

Comment:
I wonder if we should stick to `20-browsers` across the board, saves having to do the minor updates all the time 🤔